app/services/preserver/importer.rb

```diff
@@ -38,77 +31,68 @@ def initialize(metadata_file_identifier:, binary_file_identifiers:, change_set_p
     end
 
     def import!
-      fs = build_file_set(metadata_file_identifier)
-      fs_change_set = ChangeSet.for(fs)
-
       files = import_binary_nodes(binary_file_identifiers)
+      fs_change_set = ChangeSet.for(file_set)
       fs_change_set.validate(files: files)
 
-      persisted = nil
+      # Delete historic metadata because we are about to add new ones for the restored file(s)
+      fs_change_set.validate(file_metadata: [])
```

Comment:
Validating and not syncing to keep access to the old file_metadata is clever!

Reply:
Where is it used that way? It seems like everything is pulled from the old data before we get here. Is that why you made this change @eliotjordan?

Reply:
I didn't put too much thought into this, other than thinking I wanted to merge
`fs_change_set.validate(files: files)` and `fs_change_set.validate(file_metadata: [])` into a single line.

`fs_change_set.file_metadata = []` also works but #sync does not (though I'm not sure why). @hackartisan is right, I was thinking that the old file metadata is pulled out before we get here.

Reply:
So... nothing intentionally clever.